larray/inout/xw_excel.py

```diff
@@ -35,7 +36,6 @@ def is_app_alive(app):
         except Exception:
             return False
 
-
```

Comment:
the removal is strange. Those lines were added for PEP8 compliance...

Reply:
Maybe pytest-pep8 is confused because of the ``if xw is not None:``. I can add an exception for this module in particular.